### PR TITLE
Display error for checkbox lists if provided

### PR DESCRIPTION
**Pull request by ztolley**
methoderro

### CODE REVIEW

File: app/helpers/barnardos/action_view/form_helpers.rb [hunk 1]
```diff
@@ -118,6 +118,12 @@ def checkbox_group_vertical(name, legend, selection_list, values = [], error: ni
                 end
               )
             end
+            concat(
+              content_tag(:div,
+                          "#{name.to_s.humanize} #{error[0]}",
```
Comment (rgarner) [inline app/helpers/barnardos/action_view/form_helpers.rb]: The variable name says "error" but this is still an array. Can it just be a string?

**Comment by rgarner** [comment]
'k, clearly I need to alter the validator, which is spitting out some human-unfriendly stuff

**Comment by rgarner** [comment]
Am I right in saying this isn't referenced from either checkbox-using view at present?

File: app/helpers/barnardos/action_view/form_helpers.rb (hunk 1 above)
Comment (ztolley) [inline app/helpers/barnardos/action_view/form_helpers.rb]: The validators generate an array of errors. It's design to work with data passed from the validator

**Comment by ztolley** [comment]
Currently not of our pages put supply the error. I have a branch though you could look at that does for 2 screens

**Pull request merged**